boards/msba2/include/periph_conf.h

```diff
@@ -87,7 +87,31 @@ static const uart_conf_t uart_config[] = {
         .pinsel_tx   = 0,
         .pinsel_msk_rx = BIT4,
         .pinsel_msk_tx = BIT6,
-    }
+    },
+    {
+        .dev = UART1,
+        .irq_prio_rx = 6,
```

Comment:
Is there any logic or considerations needed for the IRQ priority level on these?

Reply:
Since the VIC supports 32 interrupts with 16 levels, two (or more) interrupts at the same level should be no problem.

I used 6 because that's what has been used before.